COLLADAStreamWriter/src/COLLADASWNode.cpp

```diff
@@ -17,9 +17,9 @@ namespace COLLADASW
 {
 
     //---------------------------------------------------------------
-    void Node::start()
+    void Node::start(bool forceNonInstancedNode)
     {
-        if ( !mIsInstanceNode )
+		if (!mIsInstanceNode || forceNonInstancedNode)
```

Comment:
What is ```forceNonInstancedNode``` about?
I think this is incorrect. The proxy extra belongs to instance_node